### PR TITLE
Added backup warning after PK account import

### PR DESCRIPTION
**Pull request by OlegMakarenko**
Resolved: #948

image

### CODE REVIEW

**Review by rg911 — CHANGES_REQUESTED** [review]
Nice one, 👍   could you add language files for Japanese and others?

**Comment by OlegMakarenko** [comment]
> Nice one, 👍 could you add language files for Japanese and others?

@rg911 
What placeholder for values should I use, just to copy English?

**Comment by rg911** [comment]
> > Nice one, 👍 could you add language files for Japanese and others?
> 
> @rg911
> Sure. What placeholder for values should I use, just to copy English?

For the PR review, you can use translation software (e.g. google) for now

**Comment by cryptoBeliever** [comment]
> > > Nice one, +1 could you add language files for Japanese and others?
> > 
> > 
> > @rg911
> > Sure. What placeholder for values should I use, just to copy English?
> 
> For the PR review, you can use translation software (e.g. google) for now, placeHolder should be the same for all languages,

@rg911 how you after that find messages to translate? Isn't better to use English (easier to find not translated messages)?

**Comment by OlegMakarenko** [comment]
> > > > Nice one, +1 could you add language files for Japanese and others?
> > > 
> > > 
> > > @rg911
> > > Sure. What placeholder for values should I use, just to copy English?
> > 
> > 
> > For the PR review, you can use translation software (e.g. google) for now, placeHolder should be the same for all languages,
> 
> @rg911 how you after that find messages to translate? Isn't better to use English (easier to find not translated messages)?

@rg911 @cryptoBeliever I would suggest to add some set of chars to determine which translation is mock. Let say `>>` in the beginning

File: src/language/en-US.json [hunk 1]
```diff
@@ -135,6 +135,7 @@
     "button_download": "Download",
     "button_download_paper_wallet": "Download Paper Wallet",
     "button_download_qr": "Download",
+    "button_understand_title": "Understand",
```
Comment (segfaultxavi) [inline src/language/en-US.json:138]:
```suggestion
    "button_understand_title": "Understood",
```

File: src/language/en-US.json [hunk 2]
```diff
@@ -437,6 +438,8 @@
     "mnemonic_phrase": "Mnemonic Passphrase",
     "mnemonic_qr_action_desc": "Click continue to set imported Mnemonic Words to the text area.",
     "modal_account_unlock_title": "Unlock your account",
+    "modal_backup_reminder_content": "Please note that the Private Key account cannot be restored from your mnemonic passphrase. Please backup your profile again after import is complete to include this account in the Paper Wallet",
```
Comment (segfaultxavi) [inline src/language/en-US.json:441]:
```suggestion
    "modal_backup_reminder_content": "Please note that the Private Key account cannot be restored from your mnemonic passphrase. Backup your profile again after import is complete to include this account in the Paper Wallet.",
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

File: src/language/en-US.json (hunk 1 above)
Comment (OlegMakarenko) [reply src/language/en-US.json]: Thank you 👍

**Review by bassemmagdy — APPROVED** [review]

**Review by rg911 — APPROVED** [review]

**Pull request merged**